src/scores/continuous/standard_impl.py

```diff
@@ -354,11 +354,10 @@ def pbias(
     """
     Calculates the percent bias, which is the ratio of the additive bias to the mean observed value, multiplied by 100.
 
-    Percent bias is used for evaluating and comparing forecast accuracy across stations or dataset with varying magnitudes.
-    By expressing the error as a percentage of the observed value, it allows for standardized comparisons, enabling assessment
-    of forecast performance regardless of the absolute scale of values. Like multiplicative_bias, Percent bias will return a np.inf
-    where the mean of `obs` across the dims to be reduced is 0.
-    It is defined as
+    Percent bias is used for evaluating and comparing forecast accuracy across stations or datasets with varying magnitudes.
+    By expressing the error as a percentage of the observed value, it allows for standardised comparisons, enabling assessment
+    of forecast performance regardless of the absolute scale of values. Like :py:func:`scores.continuous.multiplicative_bias`,
+    ``pbias`` will return a np.inf where the mean of ``obs`` across the dims to be reduced is 0. It is defined as
```

Comment:
```suggestion
    ``pbias`` will return a ``np.inf`` where the mean of ``obs`` across the dims to be reduced is 0. It is defined as
```
Similar to above, should "np.inf" also have two backticks on either side (so that it can also render in a grey box (I really don't know the correct term for that style of rendering))?

Reply:
Yes, I think np.inf should also be in backticks to clearly indicate it's a code concept.